### PR TITLE
calibre: fix build

### PR DESCRIPTION
**Pull request by averelld**
###### Motivation for this change

Makes calibre compile again, fixing #65313 No real testing done, but it starts again for me, and the viewer works. 
Fix maybe technically uncool, splitting qtWrapperArgs, but i don't see how to get PYTHONPATH properly interpolated otherwise. 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc @ToxicFrog

### CODE REVIEW

File: pkgs/applications/misc/calibre/default.nix [hunk 1]
```diff
@@ -49,8 +49,7 @@ stdenv.mkDerivation rec {
   ]);
 
   qtWrapperArgs = [
-    "--prefix PYTHONPATH: $PYTHONPATH"
-    "--prefix PATH: ${poppler_utils.out}/bin}"
+    "--prefix PATH : ${poppler_utils.out}/bin}"
```
Comment (lightbulbjim) [inline pkgs/applications/misc/calibre/default.nix]: What is the `}` after `bin` for?

**Comment by lightbulbjim** [comment]
Builds and runs for me on NixOS unstable.

File: pkgs/applications/misc/calibre/default.nix (hunk 1 above)
Comment (averelld) [reply pkgs/applications/misc/calibre/default.nix]: Oops, good catch, pdf conversion was broken. Fixed.

**Comment by worldofpeace** [comment]
Why are we wrapping in `installPhase`?
I don't think the proper values would even be collected at that phase. It should be `postFixup`.

**Comment by averelld** [comment]
It's been wrapped there since initial packaging 10 years ago a4473b820, I'm assuming because https://github.com/kovidgoyal/calibre/tree/master/setup looks a bit extreme.
But :+1: on an overhaul, MAGICK_INC etc look unused as well.

**Comment by volth** [comment]
Does it work without https://github.com/NixOS/nixpkgs/issues/65399 fix ?

Here it fails with
```
$ calibre
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Aborted
```

**Comment by worldofpeace** [comment]
I can fix this today :+1:

**Comment by worldofpeace** [comment]
> Does it work without #65399 fix ?
> 
> Here it fails with
> 
> ```
> $ calibre
> qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
> This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
> 
> Aborted
> ```

I checked and it appears to be wrapped correctly, though I decided to make it a little cleaner #66483

**Comment by FRidh** [comment]
Please do not use a custom builder function unless really necessary. I know it's documented as *the way* with Qt5, but we should avoid it when possible.

**Comment by worldofpeace** [comment]
> Please do not use a custom builder function unless really necessary. I know it's documented as _the way_ with Qt5, but we should avoid it when possible.

Did you mean to comment that on my PR? (context confused)

**Comment by worldofpeace** [comment]
Additionally, I think it would be appropriate for calibre. I believe it is just a qt application?

**Pull request merged**